zhaquirks/xiaomi/aqara/switch_h1.py

```diff
@@ -0,0 +1,147 @@
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.general import (
+    Alarms,
+    Basic,
+    GreenPowerProxy,
+    Groups,
+    Identify,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+
+from zhaquirks.const import (
+    ARGS,
+    ATTR_ID,
+    BUTTON,
+    CLUSTER_ID,
+    COMMAND,
+    COMMAND_DOUBLE,
+    COMMAND_HOLD,
+    COMMAND_SINGLE,
+    DEVICE_TYPE,
+    DOUBLE_PRESS,
+    ENDPOINT_ID,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    LONG_PRESS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PRESS_TYPE,
+    PROFILE_ID,
+    SHORT_PRESS,
+    VALUE,
+)
+from zhaquirks.xiaomi import (
+    LUMI,
+    BasicCluster,
+    DeviceTemperatureCluster,
+    OnOffCluster,
+    XiaomiCustomDevice,
+    XiaomiMeteringCluster,
+)
+from zhaquirks.xiaomi.aqara.opple_remote import MultistateInputCluster
+from zhaquirks.xiaomi.aqara.opple_switch import OppleSwitchCluster
+
+XIAOMI_COMMAND_SINGLE = "41_single"
+XIAOMI_COMMAND_DOUBLE = "41_double"
+XIAOMI_COMMAND_HOLD = "1_hold"
+
+
+class AqaraH1SingleRockerSwitch(XiaomiCustomDevice):
```

Comment:
Since this is a device without battery (right?), you can change `XiaomiCustomDevice` to just `CustomDevice`.

Reply:
That's correct - the device doesn't have a battery, it's fed through mains. I've implemented the change and it all works fine with it 👍